packages/tpetra/core/cmake/TpetraCore_config.h.in

```diff
@@ -2,6 +2,9 @@
 #define TPETRACORE_CONFIG_H
 /* CMake uses this file to generate TpetraCore_config.h automatically */
 
+/* define if user is using the deprecated codebase (templated ordinals, dynamicprofile) or not */
```

Comment:
See above.  "Deprecated code" is a generic thing; we should not refer to specifically deprecated features here.

Reply:
Changed.